erpnext/accounts/doctype/sales_invoice/test_sales_invoice.py

```diff
@@ -2702,6 +2702,7 @@ def test_sales_invoice_with_discount_accounting_enabled(self):
 		]
 
 		check_gl_entries(self, si.name, expected_gle, add_days(nowdate(), -1))
+		enable_discount_accounting(enable=0)
```

Comment:
@rahib-hassan this isn't required, `change_settings` decorator rolls it back to previous value after test runs.